cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -82,9 +82,8 @@ def load_network_interfaces(
             ON CREATE SET r.firstseen = timestamp()
             SET r.lastupdated = $update_tag
         WITH network_interface, netinf
-
         MERGE (acc:AWSAccount{id: $aws_account_id})
-        ON CREATE SET acc.firstseen = timestamp()
+        ON CREATE SET acc.firstseen = timestamp(), acc.inscope=true
```

Comment:
We should maybe also be intentional to delete the `foreign` property if it exists.

Reply:
Fixed in aws_account_cleanup.json job